plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/annotationProcessing.kt

```diff
@@ -32,14 +32,21 @@ fun KaptContext.doAnnotationProcessing(
     javaSourceFiles: List<File>,
     processors: List<IncrementalProcessor>,
     additionalSources: JavacList<JCTree.JCCompilationUnit> = JavacList.nil(),
-    aggregatedTypes: List<String> = emptyList()
+    typesToReprocess: List<String> = emptyList()
```

Comment:
Actually it should be types for binary classes, maybe use  `binaryTypesToReprocess` then (not sure that it's the best one)

Reply:
Done